### PR TITLE
add handler for text/plain bodies

### PR DESCRIPTION
**Pull request by AlvaroVega**
Fixes https://github.com/telefonicaid/fiware-pep-steelskin/issues/345

### CODE REVIEW

**Comment by fgalan** [comment]
I understand this PR solves the forwarding of *requests* with text/plain MIME type. In the case of Orion, this is the only one request of that case:

```
PUT /v2/entities/E/attrs/A/value
Content-Type: text/plain

80
```

But, what about of the reverse? I mean, a request without payload but that causes a *response* with text/plain. Is this PR solving that case (in negative case I'd suggest to expand it to do so). Example of what I mean:

Request:

```
GET /v2/entities/E/attrs/A/value
Accept: text/plain
```

could cause the following respones:

```
200 OK
Content-Type: text/plain

80
```

File: lib/fiware-pep-steelskin.js [hunk 1]
```diff
@@ -108,6 +108,30 @@ function xmlRawBody(req, res, next) {
     }
 }
 
+
```
Comment (fgalan) [inline lib/fiware-pep-steelskin.js]: CNR entry missing

**Comment by AlvaroVega** [comment]
Regarding with https://github.com/telefonicaid/fiware-pep-steelskin/pull/346#issuecomment-302087801 this PR covers both ways: request and responses.

**Comment by AlvaroVega** [comment]
LGTM

**Comment by fgalan** [comment]
LGTM

**Pull request merged**